### PR TITLE
Feature/57 revisit event crud

### PR DESCRIPTION
**Pull request by Ahydul**
Created more validations for event serializer and created lesson event serializer

### CODE REVIEW

File: nexong/models.py [hunk 1]
```diff
@@ -210,61 +237,83 @@ class Lesson(models.Model):
...
+    educators = models.ManyToManyField(Educator, related_name="lesson_events")
+    attendees = models.ManyToManyField(Student, related_name="lesson_events")
+    volunteers = models.ManyToManyField(Volunteer, related_name="lesson_events")
+
+
 class Event(models.Model):
     name = models.CharField(max_length=100)
     description = models.CharField(max_length=1000)
     place = models.CharField(max_length=1000)
     capacity = models.IntegerField(default=0, validators=[MinValueValidator(0)])
```
Comment (auroranavas) [inline nexong/models.py:309]: This is a mistake in the DB, sorry 😑. Please remove capacity. The logic should be like this: attendees <= max_attendees and volunteers <= max_volunteers.

**Review by auroranavas — CHANGES_REQUESTED** [review]

File: nexong/api/Event/views.py [hunk 2]
```diff
@@ -16,3 +16,15 @@ def destroy(self, request, *args, **kwargs):
         instance = self.get_object()
         self.perform_destroy(instance)
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class LessonEventApiViewSet(ModelViewSet):
+    queryset = Event.objects.all()
```
Comment (auroranavas) [inline nexong/api/Event/views.py:22]: This should be LessonEvent.

**Review by auroranavas — CHANGES_REQUESTED** [review]

**Review by auroranavas — DISMISSED** [review]
The CRUD operations and the validations work perfectly. Congratulations and keep up the good work!

**Review by pabpercab1 — DISMISSED** [review]
Looks good, and you have added all the things proposed. Well done!

**Review by JuanluRM — APPROVED** [review]
No issues found. Solid work, well done.

**Review by auroranavas — APPROVED** [review]

**Review by JuanluRM — APPROVED** [review]

**Review by pabpercab1 — APPROVED** [review]

**Pull request merged**